001-core.php

```diff
@@ -39,3 +39,17 @@ function wpcom_vip_disable_core_update_cap( $caps, $cap ) {
 	}
 	return $caps;
 }
+
+/**
+ * Diable tests in the Site Health (AKA site status) tool page
+ *
+ * By default, WordPress runs a series of tests on the Site Health tool
+	* page in wp-admin. Some tests, such as the Background Updates test, are
+	* irrelevant because the updates are managed by the VIP team.
+	* This disables all unnecessary tests.
+ */
+function remove_background_updates_test( $tests ) {
+	unset( $tests['async']['background_updates'] );
+	return $tests;
+}
+add_filter( 'site_status_tests', 'remove_background_updates_test' );
```

Comment:
Update to use prefixed and generic function name:

```suggestion
add_filter( 'site_status_tests', 'vip_disable_unnecessary_site_health_tests' );
```

Reply:
Changed 👍